### PR TITLE
Issue 16 editing category

### PR DESCRIPTION
**Pull request by BilalAlmalih**
- editing styling 
-adding functionality
-removing my learning (the file is still there I just removed it from home/page )
#16

### CODE REVIEW

File: components/Categories.jsx [hunk 1]
```diff
@@ -7,35 +7,39 @@ import StorageOutlinedIcon from "@mui/icons-material/StorageOutlined";
...
-    { name: "Astronomy", icon: <AutoAwesomeIcon /> },
+    { name: "Sales", icon: <TrendingUpIcon style={{ fontSize: 45 }} />, slug: "sales" },
+    { name: "HR", icon: <GroupsIcon style={{ fontSize: 45 }} />, slug: "hr" },
+    { name: "Drawing", icon: <GestureIcon style={{ fontSize: 45 }} />, slug: "drawing" },
+    { name: "Big Data", icon: <StorageOutlinedIcon style={{ fontSize: 45 }} />, slug: "big-data" },
+    { name: "Design", icon: <DesignServicesOutlinedIcon style={{ fontSize: 45 }} />, slug: "design" },
+    { name: "Marketing", icon: <TipsAndUpdatesIcon style={{ fontSize: 45 }} />, slug: "marketing" },
+    { name: "Astronomy", icon: <AutoAwesomeIcon style={{ fontSize: 45 }} />, slug: "astronomy" },
+    { name: "Astronomy", icon: <AutoAwesomeIcon style={{ fontSize: 45 }} />, slug: "astronomy" },
+
```
Comment (wasimtayyan) [inline components/Categories.jsx:22]: two Astronomy  !!?
you can replace it with 'web development '

**Review by wasimtayyan — CHANGES_REQUESTED** [review]
you did a great job thanks but there is a small issue you repeated the Astronomy category

File: components/Categories.jsx (hunk 1 above)
Comment (BilalAlmalih) [reply components/Categories.jsx]: i thought we were gonna change them anyway depending on the courses.

**Review by wasimtayyan — APPROVED** [review]

**Comment by vuslatsena** [comment]
There were conflicts on 2 pages @BilalAlmalih , if you solve them, we will merge. good luck, good work 🙌🏽

**Review by vuslatsena — CHANGES_REQUESTED** [review]
There were conflicts on 2 pages @BilalAlmalih , if you solve them, we will merge. good luck, good work 🙌🏽

**Review by cerenss — APPROVED** [review]
Note: If we are going to use the new course data on firebase datastore, the category names and amount will have to change.

**Review by wasimtayyan — APPROVED** [review]

**Pull request merged**